sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,689 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------
 
-from typing import List, Any, Optional, TYPE_CHECKING
+from typing import Dict, List, Any, Optional, TYPE_CHECKING, Tuple, Union
 
 from azure.core import PipelineClient
+from cryptography.hazmat.primitives import serialization
 from six import python_2_unicode_compatible
 
 if TYPE_CHECKING:
     # pylint: disable=unused-import,ungrouped-imports
     from typing import Any
 
     from azure.core.credentials import TokenCredential
-    from azure.core.pipeline.transport import HttpRequest, HttpResponse
 
 from ._generated import AzureAttestationRestClient
 from ._generated.models import (
-    AttestationType, 
-    PolicyResult as GeneratedPolicyResult, 
-    PolicyCertificatesResult, 
-    JSONWebKey, 
-    AttestationCertificateManagementBody, 
+    AttestationType,
+    PolicyResult as GeneratedPolicyResult,
+    PolicyCertificatesResult as GeneratedPolicyCertificatesResult,
+    JSONWebKey,
+    AttestationCertificateManagementBody,
     StoredAttestationPolicy as GeneratedStoredAttestationPolicy,
-    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult
+    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult,
 )
 from ._configuration import AttestationClientConfiguration
 from ._models import (
-    AttestationSigner, 
-    AttestationToken, 
-    AttestationResponse, 
-    AttestationSigningKey, 
-    PolicyCertificatesModificationResult,
-    PolicyResult,
-    AttestationTokenValidationException
+    AttestationSigner,
+    AttestationToken,
+    AttestationPolicyResult,
+    AttestationPolicyCertificateResult,
+    AttestationTokenValidationException,
 )
 import base64
 from azure.core.tracing.decorator import distributed_trace
 from threading import Lock, Thread
```

Comment:
Unused imports (`AttestationTokenValidationException` is referenced in docstrings, but as a fully qualified reference (`~azure.security.attestation.AttestationTokenValidationException`), so it doesn't need to be imported when TYPE_CHECKING or otherwise
```suggestion
)
import base64
from azure.core.tracing.decorator import distributed_trace
from threading import Lock
```